targets/TARGET_RENESAS/TARGET_RZ_A1XX/TARGET_GR_LYCHEE/trng_api_esp32.cpp

```diff
@@ -80,6 +90,10 @@ extern "C" int trng_get_bytes_esp32(uint8_t *output, size_t length, size_t *outp
     if (output_length != NULL) {
```

Comment:
If ```output_length``` is a required out parameter then ```output_length == NULL``` is an invalid input. In that case it should be tested in the beginning of the function.